src/common/guides/_openlogin-config-code-discord.mdx

```diff
@@ -1,7 +1,7 @@
 ```js
 const openloginAdapter = new OpenloginAdapter({
   adapterSettings: {
-    clientId: "YOUR-WEB3AUTH-CLIENT-ID",
+    clientId: "YOUR-WEB3AUTH-CLIENT-ID", //Optional - Provider only if you haven't provided it in the Web3Auth Instantiation Code
```

Comment:
While instantiating, it's a required field. Right? If so, we can remove this comment.

Reply:
Same applies to other references of this.

Reply:
Looking at https://github.com/Web3Auth/Web3Auth/blob/master/packages/core/src/core.ts#L31 looks like it's a required field.